profile/README.md

```diff
@@ -29,3 +29,24 @@ For other support feel free to join us here:
 - [Discord Server](https://discord.gg/7aEbB9dGRT)
 - [Reddit](https://www.reddit.com/r/pulsaredit/)
 - [Mastodon](https://fosstodon.org/@pulsaredit)
+
+
+<br>
+<br>
+
+
+
+<!---------------------------------- 🗞 News ---------------------------------->
+ 
+<pre align = center>
+ 
+<kbd align = left> <br>   <b>🗃 <a href = 'https://github.com/orgs/pulsar-edit/projects/2?pane=issue&itemId=17528685'>Backend Feature</a></b><br>   <br>   Allow package authors to transfer<br>   their ownership to another user.<br>   <br>                                                                                </kbd>     <kbd align = left> <br>   <b>🔬 Testing2</b><br>   <br>   This is a test message!<br>   <br>                                                                                </kbd><br><br><kbd align = left> <br>   <b>🔬 Testing</b><br>   <br>   This is a test message!<br>   <br>                                                                                </kbd>     <kbd align = left> <br>   <b>🎅 Merry Christmas</b><br>   <br>   from all of us here at Pulsar edit!<br>   We wish everyone a great new year!<br>   <br>                                                                                </kbd>
```

Comment:
```suggestion

```

Just gonna clear this all out to avoid commiting any incorrect messages

Reply:
Looks like I'm not able to commit just the removal. 

If you'd don't mind I'll make a commit against your PR branch to remove it, then merge the changes.

Reply:
Or it won't quite let me do that either.

Guess I'll just merge it and right away make a PR to merge the proper text.

Reply:
Yes, well by default I turn off commit access for my PRs,
since you already merged it, it doesn't matter anymore but
I can enable it if you need it next time.

Reply:
I think it has been reverted, have you looked at #118?